packages/selinux-policy/selinux-policy.spec

```diff
@@ -64,7 +64,7 @@ install -d "${moddir}"
 for m in *.cil ; do
   mod="${m%.*}"
   install -d "${moddir}/${mod}"
-  bzip2 -c "${m}" > "${moddir}/${mod}/cil"
+  install -p -m 0644 "${m}" "${moddir}/${mod}/cil"
```

Comment:
Will we consider using ZSTD?

Reply:
> Will we consider using ZSTD?

I'd like to, but probably only if it ends up as an upstream feature. The benefits from compression here are so modest that I don't think it's worth carrying an even larger patch to implement that.